docker/builder/cmd.sh

```diff
@@ -48,161 +40,54 @@ BABEL_CACHE_PATH=$(pwd)/.tmp/babel-cache/${PACKAGE}.babel.json
-      log "Connecting with Tunnel Identifier ${SC_TUNNEL_IDENTIFIER}"
-
-      set +e
-      daemon -U --name sauce_connect -- "${SC_BINARY}" \
-        -B *.wbx2.com,*.ciscospark.com,idbroker.webex.com,idbrokerbts.webex.com,127.0.0.1,localhost \
-        -t whistler.onint.ciscospark.com,internal-testing-services.wbx2.com,127.0.0.1,localhost,calendar-whistler.onint.ciscospark.com \
-        -vv \
-        -l "$(pwd)/reports/sauce/sauce_connect.$(echo "${PACKAGE}" | awk -F '/' '{ print $NF }').${SC_ITERATION}.log" \
-        --tunnel-identifier "${SC_TUNNEL_IDENTIFIER}" \
-        --pidfile "${SC_PID_FILE}" \
-        --readyfile "${SC_READY_FILE}"
-      EXIT_CODE=$?
-      set -e
-
-      if [ "${EXIT_CODE}" -ne "0" ]; then
-        log "Failed"
-        continue
-      fi
-
-      log "Succeeded, waiting for readyfile"
-
-      READY_CHECK=0
-      while [[ ! -e "${SC_READY_FILE}" && ${READY_CHECK} -lt 24 ]]; do
-        sleep 5
-        let READY_CHECK+=1
-        log "Ready Check: ${READY_CHECK}"
-      done
-
-      if [ ! -e "${SC_READY_FILE}" ]; then
-        log "Ready Check Failed"
-        set +e
-        stopSauce
-        set -e
-        continue
-      fi
-
-      break;
-    done
-
-    if [ -e "${SC_PID_FILE}" ]; then
-      log "SC: Connected with Tunnel Identifier ${SC_TUNNEL_IDENTIFIER}"
-    else
-      log "SC: Failed to connect to Sauce Labs"
-      exit 10
-    fi
-  fi
-
-  set +e
-  echo "" >> "${GRUNT_LOG_FILE}"
-  echo "### Attempt ${SUITE_ITERATION} ###" >> "${GRUNT_LOG_FILE}"
-  echo "" >> "${GRUNT_LOG_FILE}"
-  if [ "${PACKAGE}" == "samples" ]; then
+set +e
+if [ "${PACKAGE}" == "samples" ]; then
     CI=true npm run test:samples >> "${GRUNT_LOG_FILE}" 2>&1
     EXIT_CODE=$?
     # Generate the coverage report
     npm run tooling -- test --no-tests
-  elif [ "${PACKAGE}" == "@webex/webex-server" ]; then
+    elif [ "${PACKAGE}" == "@webex/webex-server" ]; then
     npm test -- --package @webex/webex-server --node >> "${GRUNT_LOG_FILE}" 2>&1
     EXIT_CODE=$?
     # Generate the coverage report
     npm run tooling -- test --no-tests --node
-  else
+else
     # Skip unit tests in gating pipelines
     if [ -n "${PIPELINE}" ]; then
-      npm run test -- --no-unit >> "${GRUNT_LOG_FILE}" 2>&1
+        npm run test -- --no-unit >> "${GRUNT_LOG_FILE}" 2>&1
     else
-      npm run test >> "${GRUNT_LOG_FILE}" 2>&1
+        npm run test >> "${GRUNT_LOG_FILE}" 2>&1
     fi
     EXIT_CODE=$?
-  fi
-  set -e
+fi
+set -e
 
-  # No need to repeat if there was a success
-  if [ "${EXIT_CODE}" -eq "0" ]; then
+# No need to repeat if there was a success
+if [ "${EXIT_CODE}" -eq "0" ]; then
     exit 0
-  fi
-
-  if [ -z "${SAUCE_IS_DOWN}" ]; then
-    if [ ! -e "${SC_PID_FILE}" ]; then
-      log "SC_PID_FILE missing after suite exited with ${EXIT_CODE}"
-      rm -f "${SC_READY_FILE}"
-      continue
-    fi
-
-    PID=$(cat "${SC_PID_FILE}")
-
-    if ps -p "${PID}" > /dev/null; then
```

Comment:
Need to replace this.
I was thinking a timeout of some kind that exists out after 15 minutes if the container doesn't exit out automatically